File: C++/20.cpp
#include <iostream>

using namespace std;

int main()
{

    int number;
    cout << "Enter the number: " << endl;
    cin >> number;

    int firstDigit = 0, lastDigit = number % 10;

    while (number > 0)
    {
        firstDigit = number % 10;
        number = number / 10;
    }

    cout << "Sum of the elements: " << firstDigit + lastDigit;
}

File: Python/20.py
def sum_first_and_last_digit(number):

    numberStr = str(abs(number))
    firstDigit = int(numberStr[0])  
    lastDigit = int(numberStr[-1]) 
    return firstDigit + lastDigit


number = int(input("Введите число: "))
result = sum_first_and_last_digit(number)
print(f"Сумма первой и последней цифры числа {number} = {result}")
